axelor-account/src/main/java/com/axelor/apps/account/service/move/massentry/MassEntryMoveCreateServiceImpl.java

```diff
@@ -128,12 +129,12 @@ public Move generateMassEntryMove(Move move) throws AxelorException {
         moveLineComputeAnalyticService.generateAnalyticMoveLines(newMoveLine);
         moveLineMassEntryRecordService.setAnalytics(newMoveLine, moveLine);
         if (ObjectUtils.notEmpty(moveLine.getAnalyticMoveLineList())) {
-          moveLine
-              .getAnalyticMoveLineList()
-              .forEach(
-                  analyticMoveLine ->
-                      newMoveLine.addAnalyticMoveLineListItem(
-                          analyticMoveLineRepository.copy(analyticMoveLine, false)));
+          newMoveLine.clearAnalyticMoveLineList();
+          for (AnalyticMoveLine analyticMoveLine : moveLine.getAnalyticMoveLineList()) {
+            AnalyticMoveLine copyAnalyticMoveLine =
+                analyticMoveLineRepository.copy(analyticMoveLine, false);
+            newMoveLine.addAnalyticMoveLineListItem(copyAnalyticMoveLine);
```

Comment:
I think you don't have to store `copyAnalyticMoveLine`

Reply:
Resolved